Add NV LTC scraper

zachlipton opened the pull request.

zachlipton committed:
Add NV LTC scraper

actions-user committed:
Updating scraped LTC data

julia326 commented on data-collection-scripts/ltc-scraper/LICENSE:
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
why its own here vs the repo-wide license? (Ditto on the .gitignore contents, could add to the repo-wide one)

julia326 commented on data/ltc_nv.csv:
```diff
@@ -0,0 +1,178 @@
+Facility Name,Facility Type,County,Unique Confirmed Cases,Recovered Residents/Patients,Recovered Staff,Deaths Residents/Patients,Deaths Staff,Attack Rate Residents/Patients,Attack Rate Staff,Confirmed COVID-19 Cases Residents/Patients,Confirmed COVID-19 Cases Staff,Confirmed COVID-19 Cases Imported
```
I think this is a copy of the `nv_ltc.csv` above?

zachlipton committed:
Some cleanup from the repository move

zachlipton commented:
Ah good catch. A few things were leftover from the repository move. 

I deleted the separate license, consolidated .gitignore, and removed the extra data directory (the GH Actions job will write to the repo-level data directory)

julia326 commented on .github/workflows/update_ltc.yml:
```diff
@@ -0,0 +1,48 @@
+            node-version: '14.x'
+        - name: Install dependencies
+          run: npm install
+        # there's an extremely irritating DNS bug somewhere (https://github.com/actions/virtual-environments/issues/798)
```
hahah wow

zachlipton replied:
It’s not DNS
There is a no way it’s DNS
It was DNS

Do not get me started. It was extraordinarily annoying

julia326 commented on data-collection-scripts/ltc-scraper/.gitignore:
```diff
@@ -0,0 +1,106 @@
+# Logs
```
this one's still here?

zachlipton replied:
Just a sec. Fighting with the GH Actions commit action, but I'll make sure it's gone

julia326 reviewed (approved):
Good to merge once you remove the extra .gitignore file :)
